src/%IPM/Common.inc

```diff
@@ -6,10 +6,10 @@ ROUTINE %IPM.Common [Type=INC]
 #define ZPMStudioDocumentModule %ZPMStudioDocumentModule
 
 #; Local % Variable to track modules already handled for a given end goal
-#define ZPMHandledModules %ZZPMHandledModules
+#define ZPMHandledModules %ZPMHandledModules
```

Comment:
Create a Jira to change all these macros and their values to be IPM + use ^IRIS.Temp instead of ^CacheTemp

Reply:
@isc-kiyer  https://usjira.iscinternal.com/browse/HSIEO-6261 Jira created here!